app/drawer.js

```diff
@@ -2519,7 +2519,7 @@
     brush.layer = uiLayer;
     brush.segments = brushSegments;
     brush.pivot = new Point(brushSize / 2 - 0.5, brushSize / 2 - 0.5);
-    brush.position = getBrushCenteredCoordinate(prevPos);
+    brush.position = prevPos;
```

Comment:
A slightly better fix would probably be:
`brush.position = getBrushCenteredCoordinate(prevPosOutline);`
Then the prevPos variable could be removed

I use getBrushCenteredCoordinate to make sure the brush preview is always aligned to the grid.

Thanks for finding and fixing this bug!

Reply:
I tried that that version instead (using `getBrushCenteredCoordinate()`), however it causes flickering. Even odd-sized brush sizes push the overlay a half cell to the bottom right, and then it snaps back to center for even-sizes.

I think it feels better (and still grid aligned) without the transform (just using `prevPos`). What do you think? Can you try both and see?

Reply:
I tried both - the issue is that if you don't use getBrushCenteredCoordinate, if you click you will find that the overlay doesn't always match up with what is drawn. You will also notice that if you move your mouse, the overlay will jump to a new position. It will flicker and jump around as you change the brush size, but it will always match what you are about to draw.

Reply:
Alright, sounds reasonable then. I made the change already so this should be ready to merge.